src/strings/i18n.js

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import english from './english'
+
+i18n
+  .use(XHR)
+  .use(LanguageDetector)
+  .init({
+    fallbackLng: 'en',
+    debug: true,
```

Comment:
Set this to `process.env.NODE_ENV !== 'production'` so we skip logging in production.